frontend/syntaxmeets/src/components/SyntaxEditor/SyntaxEditor.js

```diff
@@ -85,6 +86,89 @@ const SyntaxEditor = (props) => {
     setPopup(true);
   };
 
+  const uploadFile = () => {
+    document.querySelector("#upload").click();
+  }
+
+  const handleInputChange = (newInput) => {
+    setCodeInput(newInput);
+  };
```

Comment:
Also please remove that, as we are handling this part in separate component `INPUT`